draft-ietf-quic-multipath.md

```diff
@@ -251,6 +251,9 @@ If endpoint receives an unexpected value for the transport parameter
 TRANSPORT_PARAMETER_ERROR (specified in {{Section 20.1 of QUIC-TRANSPORT}})
 and close the connection.
 
+The enable_multipath parameter MUST NOT be stored for 0-RTT resumption as
+new paths can only be used after handshake completion.
```

Comment:
Maybe something like:
```suggestion
The enable_multipath parameter MUST NOT be remembered
({{Section 7.4.1 of QUIC-TRANSPORT}}).
New paths can only be used after handshake completion.
```

I think it would be better to follow the term ("remembered") used in RFC 9000.